### ISSUE TITLE
"Delete annotation" option

### ISSUE THREAD

**Issue by ChiaraPalladino** [pelagios/recogito2#74]
Not sure if the "cancel" button is supposed to serve this purpose too, but currently there is no way to delete an annotation (am I right?)

**Comment by rsimon** [comment]
Indeed, it's a bit tricky in terms of UI. Happy to hear input from what would be the best workflow!

In general, the annotation gets deleted as soon as the last 'body' (comment, place, etc.) is removed __and__ OK is clicked. (At least this way it's always possible to cancel.) But like in case of the place verification (issue #73) this should probably be more instant.
